server/sonar-server/pom.xml

```diff
@@ -176,6 +176,10 @@
           <groupId>com.carrotsearch.randomizedtesting</groupId>
           <artifactId>randomizedtesting-runner</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
given the title of the first commit, I would expect to see only the change on `server/sonar-server/pom.xml` in it
It seems the other changes belong to the last commit

Reply:
it's on purpose. I agree that the commit title can be misleading. securemock overrode mockito in some cases like:
- signature of ArgumentMatcher implementations (Object -> T). That's the reason why they have been fixed and replaced by lambda
- any(Class) supported null values in securemock but not in latest version of mockito. Replacing calls by any() fixes the issue.